optimum/exporters/tasks.py

```diff
@@ -1011,6 +1013,8 @@ def get_exporter_config_constructor(
                 The model type to retrieve the config for.
             model_name (`Optional[str]`, defaults to `None`):
                 The name attribute of the model object, only used for the exception message.
+            exporter_config_kwargs( `Optional[Dict[str, Any]]`, defaults to `None`):
```

Comment:
just for my knowledge, why is this needed? I see it is only used in `test_tflite_export.py`.

Reply:
It is used to provide arguments to the config initalizer when using the `TasksManager`.

Reply:
Why is it required for tflite and not onnx export?

Reply:
To provide axes values.